src/react/components/_finances/FinancialResources/SummerSessions.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import APILink from 'React/components/APILink';
+import HasAccessTo from './HasAccessTo';
+import FinancialResourcesCategoryHeader from './FinancialResourcesCategoryHeader';
+import './FinancialResources.scss';
+
+const SummerSessions = ({ links, getLink, expanded, sirStatus }) => {
+  const [isExpanded, setIsExpanded] = useState(expanded);
+
+  const isUndergradSir =
+    sirStatus.sirStatuses !== undefined || sirStatus.sirStatuses !== null
+      ? false
+      : sirStatus.sirStatuses.some(sirStatus => {
```

Comment:
Reverse the order of ternary returns to simplify the predicate to `sirStatus.sirStatuses`.